qiskit/providers/ibmq/api_v2/rest/schemas/job.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Schemas for job."""
+from marshmallow.validate import OneOf
+from qiskit.providers.ibmq.apiconstants import ApiJobStatus
+from qiskit.validation import BaseSchema
+from qiskit.validation.fields import String, Url, Nested, Integer
+
+
+# Helper schemas.
+
+class InfoQueueResponseSchema(BaseSchema):
+    """Nested schema for StatusResponseSchema"""
+
+    # Optional properties
+    position = Integer(required=False, missing=0)
+    status = String(required=False, missing=False)
```

Comment:
The default status, of type `String`, is `False`? I think you misread the code. It assigns the default return value (_not_ the default `queue_status`) to False.

Reply:
Totally right, I rushed this change. Fixed!